Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamerBase.cpp

```diff
@@ -1276,12 +1274,18 @@ void MediaPlayerPrivateGStreamerBase::handleProtectionStructure(const GstStructu
         return;
     }
 
-    String keySystemUUIDString = keySystemUUID ? keySystemUUID.get() : "(unspecified)";
     InitData initData(mappedInitData.data(), mappedInitData.size());
-    GST_TRACE("init data encountered for %s of size %" G_GSIZE_FORMAT " with MD5 %s", keySystemUUIDString.utf8().data(), mappedInitData.size(), GStreamerEMEUtilities::initDataMD5(initData).utf8().data());
-    GST_MEMDUMP("init data", mappedInitData.data(), mappedInitData.size());
+    initializationDataEncountered(initData, keySystemUUID ? keySystemUUID.get() : "(unspecified)");
```

Comment:
Since we're refactoring, the whole keySystemUUID stuff is unnecessary and could be removed from the structures imo

Reply:
We could, yes, might be worth doing if we move this upstream :wink:

Reply:
Oh, the comment was outdated, I guess I tackled what you mean :sunglasses: